jar-service/src/main/java/com/sixsq/slipstream/run/RuntimeParameterResource.java

```diff
@@ -218,4 +262,14 @@ private void logTimeDiff(String msg, long before, long after) {
 	protected void logTimeDiff(String msg, long before) {
 		logTimeDiff(msg, before, System.currentTimeMillis());
 	}
+
+	private boolean isGlobalStateParameter() { return RuntimeParameter.GLOBAL_STATE_KEY.equals(key); }
```

Comment:
I recall IntelliJ separated, but I recall reverting it for some reason :-/.  I'm separating onto different lines now.  Thanks.